metadata_backend/conf/workflows/1_bigpicture.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "bigpicture",
+    "steps": [
+        {
+            "title": "Study, DAC and Policy",
+            "required": true,
+            "schemas": [
+                {
+                    "name": "study",
+                    "required": true,
+                    "hide": []
+                },
+                {
+                    "name": "dac",
+                    "required": true,
+                    "hide": []
+                },
+                {
+                    "name": "policy",
+                    "required": true,
+                    "hide": []
+                }
+            ]
+        },
+        {
+            "title": "Files",
+            "required": false,
+            "schemas": []
+        },
+        {
+            "title": "Describe",
+            "required": true,
+            "schemas": [
+                {
+                    "name": "sample",
+                    "required": false,
+                    "hide": []
+                },
+                {
+                    "name": "experiment",
+                    "required": false,
+                    "hide": []
+                },
+                {
+                    "name": "analysis",
+                    "required": false,
+                    "hide": []
+                },
+                {
+                    "name": "run",
+                    "required": false,
+                    "hide": []
+                },
+                {
+                    "name": "dataset",
+                    "required": true,
+                    "hide": []
+                }
+            ]
+        }
+    ],
+    "publish": [
```

Comment:
I agree that it's a step, but it's also a different kind of step. The other steps don't need to call endpoints. We could also make it so that the `publish` would be an object, and we would have a field for dependencies, so it would require other schemas.

Reply:
sounds ok